pipeline/beam_tables.py

```diff
@@ -372,6 +374,22 @@ def _get_full_table_name(self, table_name: str) -> str:
     """
     return self.project + ':' + table_name
 
+  def _compare_sets(self, table_set: set, gcs_set: set) -> None:
+    if table_set != gcs_set:
+      table_diff = table_set.difference(gcs_set)
+      diff_exception = ''
+      if table_diff:
+        diff_exception += '\nThe following files are missing in BigQuery:\n'
+        missing_files = ('\n').join(sorted(table_diff))
+        diff_exception += missing_files
+      gcs_diff = gcs_set.difference(table_set)
+      if gcs_diff:
+        diff_exception += '\nThe following files are missing in the GCS folder:\n'
```

Comment:
Assuming that this currently fails when running a pipeline, can you add a comment/pipeline link in the PR description explaining the current failure? (No need to fix the files)